packages/block-library/src/post-content/edit.js

```diff
@@ -13,28 +14,45 @@ import {
 	Warning,
 } from '@wordpress/block-editor';
 import { useEntityProp, useEntityBlockEditor } from '@wordpress/core-data';
+import { useMemo } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import { useCanEditEntity } from '../utils/hooks';
 
-function ReadOnlyContent( { userCanEdit, postType, postId } ) {
+function ReadOnlyContent( { layout, userCanEdit, postType, postId } ) {
 	const [ , , content ] = useEntityProp(
 		'postType',
 		postType,
 		'content',
 		postId
 	);
 	const blockProps = useBlockProps();
+
+	const themeSupportsLayout = useSelect( ( select ) => {
```

Comment:
Since now both `EditableContent` and `ReadOnlyContent` use the `layout` we can remove this logic from both and add it only to `Content`.

Reply:
Thanks, I've moved the layout logic up to the Content component 👍